eslint-config-opent2t/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "eslint-config-opent2t",
```

Comment:
>eslint-config-opent2t [](start = 11, length = 21)

shouldn't this start with 'opent2t', per our convention.. ? 
opent2t-eslint-config